### ISSUE TITLE
Support for Mobile

### ISSUE THREAD

**Issue by kevzlou7979** [riktar/jkanban#59]
Hi guys,

This is an awesome library, just want to ask if the kanban supports Mobile especially Drag and Drop. See the image below (Nevermind the offline support ) Just want to show you below how Mobile kanban works.

![MobileOffline2](https://user-images.githubusercontent.com/3138071/64505232-bb2b8700-d305-11e9-89a1-22f9f5090402.gif)

**Comment by marcosrocha85** [comment]
jKanban uses Dragula in order to make drag and drop possible. Dragula (and jKanban) does supports mobile apps, see:
![jKanban](https://media.giphy.com/media/U8AJbdwtYSNqnc3T3Q/source.gif)

**Comment by kevzlou7979** [comment]
Yeah but the horizontal scroll seems broken when we drag the item to another board.

**Comment by kevzlou7979** [comment]
Same with https://github.com/riktar/jkanban/issues/18
